packages/@expo/cli/src/start/server/middleware/metroOptions.ts

```diff
@@ -46,11 +46,13 @@ export type ExpoMetroOptions = {
   runModule?: boolean;
 };
 
+// See: @expo/metro-config/src/serializer/fork/baseJSBundle.ts `ExpoSerializerOptions`
 export type SerializerOptions = {
   includeSourceMaps?: boolean;
   output?: 'static';
   splitChunks?: boolean;
   usedExports?: boolean;
+  isExporting?: boolean;
```

Comment:
Rename to `exporting` because these are sometimes passed as query parameters and camel case makes me nervous here.